modules/nf-core/plink/recode/meta.yml

```diff
@@ -0,0 +1,145 @@
+name: "plink_recode"
+description: Recodes plink bfiles into a new text fileset applying different modifiers
+keywords:
+  - recode
+tools:
+  - "plink":
+      description: "Whole genome association analysis toolset, designed to perform a range of basic, large-scale analyses in a computationally efficient manner."
+      homepage: "https://www.cog-genomics.org/plink"
+      documentation: "https://www.cog-genomics.org/plink/1.9/data#recode"
+      tool_dev_url: "https://www.cog-genomics.org/plink/1.9/dev"
+      doi: ""
+      licence: "['GPL']"
+
+input:
+  # Only when we have meta
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  #
+  - bed:
+      type: file
+      description: PLINK binary biallelic genotype table file
+      pattern: "*.{bed}"
+  - bim:
+      type: file
+      description: PLINK extended MAP file
+      pattern: "*.{bim}"
+  - fam:
+      type: file
+      description: PLINK sample information file
+      pattern: "*.{fam}"
+
+output:
```

Comment:
Could you maybe specify which arguments are needed to get the different output files?